scripts/github-status/github-status.rb

```diff
@@ -142,7 +146,7 @@ def show_forcerebuild_pull_requests
 
 optparse.parse!
 
-ghc=GHClientHandler.new()
+ghc=GHClientHandler.new(config = { repository: options[:repository] })
```

Comment:
BTW I think you could also write

    ghc=GHClientHandler.new(repository: options[:repository])

if you prefer.